homeassistant/components/asuswrt/__init__.py

```diff
@@ -1,18 +1,43 @@
 """Support for ASUSWRT devices."""
 
+import logging
+
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import EVENT_HOMEASSISTANT_STOP, Platform
 from homeassistant.core import Event, HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
 
-from .const import DATA_ASUSWRT, DOMAIN
+from .bridge import migrate_legacy_protocols
+from .const import DATA_ASUSWRT, DOMAIN, FLOW_VERSION
 from .router import AsusWrtRouter
 
 PLATFORMS = [Platform.DEVICE_TRACKER, Platform.SENSOR]
 
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug("Migrating from version %s", entry.version)
+
+    if entry.version != FLOW_VERSION:
+        if (new_conf := await migrate_legacy_protocols(hass, dict(entry.data))) is None:
+            return True
+
+        entry.version = FLOW_VERSION
+        if new_conf:
+            hass.config_entries.async_update_entry(entry, data=new_conf)
+
+        _LOGGER.info("Migration to version %s successful", entry.version)
```

Comment:
info seems way to high for this?

Reply:
I can set `debug`, but this will occur only once when migration is completed. We don't want provide this information?

Reply:
If it's successful it's not necessary to log it.  At least other integrations tend not do, from what I've read in the code.

Reply:
Changed to debug level